### PR TITLE
add 0.12 to Travis Testing

### PR DESCRIPTION
**Pull request by vladikoff**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 language: node_js
 node_js:
   - "0.10"
+  - "0.12"
```
Comment (jrgm) [inline .travis.yml]: can you add iojs (no need for distint 1.x and 2.x though)

Comment (pdehaan) [inline .travis.yml]: @jrgm: Are you saying use:
```yaml
 - "iojs-v2"
```

Or just something like this?:
```yaml
 - "iojs"
```

**Comment by jrgm** [comment]
The latter. Just track latest util it matters.

**Comment by rfk** [comment]
LGTM but looks like something went wrong with the travis build?

**Pull request merged**